lib/ci-integrations/circle-ci.js

```diff
@@ -0,0 +1,25 @@
+const exec = require('child_process').execSync
+const _ = require('lodash')
+
+const env = process.env
+
+function test () {
+  return env.CIRCLECI === 'true'
```

Comment:
I kinda like the idea to have a single file that contains all tests. We could then run the tests and only require the mapping of the CI service we're actually running on.

I know that the getValues function is only executed on demand, but that way we could avoid loading the mappings' dependencies into memory. We could then be more liberal about what individual CI services can `require` because it's not harming users on other platforms.

What do you think? Does it make sense, or is it an unnoticeable optimization?